spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Smoke test for demo app', function() {
+    
+    browser.waitForAngularEnabled(false);
+    browser.manage().window().maximize();
+    browser.get('http://automationpractice.com/index.php?controller=authentication&back=my-account');
```

Comment:
You should first navigate to home page (http://automationpractice.com/index.php) , then click Sign In link.
This is from your precondition in test cases document.